controlpanel/settings/common.py

```diff
@@ -555,3 +555,9 @@
 AIRFLOW_SECRET_KEY = os.environ.get('AIRFLOW_SECRET_KEY')
 AIRFLOW_FERNET_KEY = os.environ.get('AIRFLOW_FERNET_KEY')
 
+
+# -- Slack
+SLACK = {
+    "api_token": os.environ.get('SLACK_API_TOKEN'),
```

Comment:
If this is required (it is) I'd use `[]` instead of `get()` so that we blow up early if misconfigured instead of having a broken CP running and raising exceptions.